<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VolumeProfile</class>
 <widget class="QWidget" name="VolumeProfile">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>556</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Volume profile</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <widget class="QWidget" name="formLayoutWidget">
      <layout class="QFormLayout" name="formLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="startDateLabel">
         <property name="text">
          <string>Start date</string>
         </property>
         <property name="buddy">
          <cstring>cb_start</cstring>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="cb_start"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="endDateLabel">
         <property name="text">
          <string>End date</string>
         </property>
         <property name="buddy">
          <cstring>cb_end</cstring>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="cb_end"/>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="numOfBinsLabel">
         <property name="text">
          <string>Num of bins</string>
         </property>
         <property name="buddy">
          <cstring>cb_num</cstring>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QComboBox" name="cb_num"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="btn_add">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btn_delete">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="tableView"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
